### ISSUE TITLE
Enrich the documentation with  IWOCL & SYCLcon talk about SSCP. 

### ISSUE THREAD

**Issue by Momellouky** [AdaptiveCpp/AdaptiveCpp#1034]
Hi, 

If I may, i find useful to add to the documentation a link to the talk entitled "The First Single Pass SYCL Compiler with Unified Code Representation Across Backends" in the section about generic SSCP compilation flow. In my point of view, the talk explains in further details what the documentation tries to say. 

However, if you would love to add the link mentioned above and you don't have enough time, just assign it to me. 

Cheers,

**Comment by illuhad** [comment]
Good idea. Maybe also link the paper, which contains some more details? We already have the paper listed here: https://github.com/OpenSYCL/OpenSYCL#citing-open-sycl

So maybe just add the video link there, and then reference that section from the documentation on the SSCP compilation flow? What do you think?

If you'd like to create a PR, please go ahead :)

**Comment by Momellouky** [comment]
Hi, 

Yes, it is interesting to reference also the paper. I will add the video link, and I will reference the paper in the SSCP compilation flow.
